MAINTAINERS.yml

```diff
@@ -1155,6 +1155,19 @@ Release Notes:
   labels:
     - "area: PWM"
 
+"Drivers: SDHC":
+  status: maintained
+  maintainers:
+    - danieldegrasse
+  files:
+    - drivers/sdhc/
+    - tests/drivers/sdhc/
+    - include/zephyr/drivers/sdhc.h
+    - dts/bindings/sdhc/
+    - doc/hardware/peripherals/sdhc.rst
+  labels:
+    - "area: Disk Access"
```

Comment:
might be helpful to add a dedicated label for this area, but no strong opinion.

Reply:
I think that is a good idea, but I don't know what is required. Do I just need to change the YAML to `- "area: SD"`, or do I need to somehow create a label in the github interface as well?

Reply:
Yes, changing the YAML here should suffice.